src/main/java/lotto/domain/RANK.java

```diff
@@ -0,0 +1,37 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
+public enum RANK {
```

Comment:
```suggestion
public enum Rank {
```
클래스명은 대문자로 시작하는 camel case로 작성해주세요 :)